app/addons/verifyinstall/resources.js

```diff
@@ -11,152 +11,159 @@
   },
-
-  testView: function () {
-    var deferred = FauxtonAPI.Deferred();
-    var promise = $.get(viewDoc.url() + '/_view/testview');
-
-    promise.then(function (resp) {
-      resp = _.isString(resp) ? JSON.parse(resp) : resp;
-      var row = resp.rows[0];
-      if (row.value === 6) {
-        return deferred.resolve();
-      }
-      var reason = {
-        reason: 'Values expect 6, got ' + row.value
-      };
-
-      deferred.reject({responseText: JSON.stringify(reason)});
-    }, deferred.reject);
-
-    return deferred;
+  testView: function() {
+    return json(viewDoc.url() + "/_view/testview").then(
+      function(resp) {
+        var row = resp.rows[0];
+        if (row.value === 6) {
+          return Promise.resolve();
+        }
+        var reason = {
+          reason: "Values expect 6, got " + row.value
+        };
+        return Promise.reject({ responseText: JSON.stringify(reason) });
+      },
+      Promise.reject
+    );
   },
+  setupView: function() {
+    var doc1 = new Documents.Doc({ _id: "test_doc_10", a: 1 }, {
+      database: db
+    });
+    var doc2 = new Documents.Doc({ _id: "test_doc_20", a: 2 }, {
+      database: db
+    });
+    var doc3 = new Documents.Doc({ _id: "test_doc_30", a: 3 }, {
+      database: db
+    });
 
-  setupView: function () {
-    var doc1 = new Documents.Doc({_id: 'test_doc_10', a: 1}, { database: db });
-    var doc2 = new Documents.Doc({_id: 'test_doc_20', a: 2}, { database: db });
-    var doc3 = new Documents.Doc({_id: 'test_doc_30', a: 3}, { database: db });
-
-    viewDoc = new Documents.Doc({
-      _id: '_design/view_check',
-      views: {
-        'testview': {
-          map: 'function (doc) { emit(doc._id, doc.a); }',
-          reduce: '_sum'
+    viewDoc = new Documents.Doc(
+      {
+        _id: "_design/view_check",
+        views: {
+          testview: {
+            map: "function (doc) { emit(doc._id, doc.a); }",
+            reduce: "_sum"
+          }
         }
+      },
+      {
+        database: db
       }
-    }, {
-      database: db
-    });
+    );
 
-    return FauxtonAPI.when([doc1.save(), doc2.save(), doc3.save(), viewDoc.save()]);
+    return FauxtonAPI.when([
+      doc1.save(),
+      doc2.save(),
+      doc3.save(),
+      viewDoc.save()
+    ]);
   },
-
-  setupReplicate: function () {
+  setupReplicate: function() {
     return $.ajax({
-      url: app.host + '/_replicate',
-      contentType: 'application/json',
-      type: 'POST',
-      dataType: 'json',
+      url: app.host + "/_replicate",
+      contentType: "application/json",
+      type: "POST",
+      dataType: "json",
       processData: false,
       data: JSON.stringify({
         create_target: true,
-        source: 'verifytestdb',
-        target: 'verifytestdb_replicate'
+        source: "verifytestdb",
+        target: "verifytestdb_replicate"
       })
     });
   },
-
-  testReplicate: function () {
+  testReplicate: function() {
```

Comment:
These changes don't really seem related to your PR. It just makes it a little harder to review everything